core/src/main/java/com/novoda/noplayer/internal/mediaplayer/InternalMediaPlayerTypeReader.java

```diff
@@ -2,7 +2,7 @@
 
 import android.os.Build;
 
-public class PlayerChecker {
+class InternalMediaPlayerTypeReader {
```

Comment:
I wasn't sure, because this is the internal media players internal player (nu player/awesome player) 🎃

happy to change